content/en/device-support/_index.md

```diff
@@ -10,32 +10,38 @@ weight: 100
-- MacBooks using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
+- iPhones and iPads using the same Apple ID (synced automatically)
+- Macs using the same Apple ID (synced automatically)
+- Macs using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
 - Windows devices in Edge and Chrome using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
-- Chrome OS devices using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
+- Chromebooks and other Chrome OS devices using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
 - Ubuntu devices in Edge and Chrome using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
 
 Passkeys created in **Android** can be used on:
 
 - The same Android device
 - Android devices using the same Google account (synced automatically)
-- MacBooks using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
+- Macs using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
 - Windows devices in Edge and Chrome using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
 - iPhones and iPads using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
-- Chrome OS devices using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
+- Chromebooks and other Chrome OS devices using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
 - Ubuntu devices in Edge and Chrome using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
 
 Passkeys created in **macOS** can be used on:
 
-- MacBooks using the same Apple ID and iCloud Keychain (synced automatically)
-- iPhones and iPads using the same Apple ID and iCloud Keychain (synced automatically)
+- Macs using the same Apple ID (synced automatically)
+- iPhones and iPads using the same Apple ID (synced automatically)
+  - ...which can then be used on:
+    - Windows devices in Edge and Chrome using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
+    - Chromebooks and other Chrome OS devices using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
+    - Ubuntu devices in Edge and Chrome using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
+    - Macs signed in with a different Apple ID, using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
+    - iPhones and iPads signed in with a different Apple ID, using [FIDO Cross-Device Authentication](/docs/reference/terms/)
```

Comment:
The way these bullet points are nested here gives the impression that only passkeys created in macOS can be used for cross-device authentication...I think these "possibilities of CDA" need their own section because this all applies to passkeys accessible on Android devices too.

Reply:
The intent is to show that passkeys created on macOS, that are accessible on an iOS device signed into the same account, can then be used on those other devices via CDA.

Would tweaking the middle tier bullet help? I don't understand the comment about Android.

Reply:
see [c1c065f](https://github.com/passkeydeveloper/passkeys.dev/pull/184/commits/c1c065f24e34e71f3d1db820e4873090041843b1)

Reply:
Looks good to me!